tcpproxy.go

```diff
@@ -93,11 +94,19 @@ func equals(want string) Matcher {
 
 // config contains the proxying state for one listener.
 type config struct {
-	routes      []route
+	sync.Mutex  // protect r/w of routes
```

Comment:
comment should be like:

    mu sync.Mutex // guards following
    x int
    y int

 and blank line before stuff that is no longer guarded by it.

Reply:
Done.